src/sinol_make/__init__.py

```diff
@@ -30,6 +32,20 @@ def main():
 
 	for command in commands:
 		if command.get_name() == args.command:
+			if sys.platform == 'linux' and not util.check_oiejq():
+				print(util.warning('`oiejq` is not installed. It will be installed now.'))
+
+				try:
+					if util.install_oiejq():
+						print(util.info('`oiejq` was successfully installed.'))
+					else:
+						print(util.error('`oiejq` could not be installed. You can try installing it manually.'))
```

Comment:
This error message won't help the user, unless we tell him how to manually install it. We can at least tell from where to get oiejq and that it's possible to pass oiejq path through a CLI argument.

Reply:
1a2ba408e2689a8680f91b87a38cd36c7b7647d4